### PR TITLE
Multi page site

### PR DESCRIPTION
**Pull request by RickyAvina**
I have changed the layout from a one-page site (which is in the branch "Web_Branch" to a multipage site) because the one page site was very limiting in that the website felt crowded and there was not much flexibility in the layout of each topic.

I have finished my main landing page and my projects pages, the contact page is still yet to be finished, but below are screenshots detailing the UI.

**Landing Page**
- The main landing page looks like this (https://prnt.sc/t5yhpv) Right now, this is a filler image.
- The landing page has parallax scrolling that reveals a small bio: (https://prnt.sc/t5yivi)

**Projects Page**
- There are cards with animations that act as links to my projects (https://prnt.sc/t5yj61)

Here are the individuals projects (4 for now)

**HoloPed**
- The project pages also have parrallax scrolling, this particular background is a gif, which is a working demo of the project (https://prnt.sc/t5yjzb)
-  These are the details of the project, each section has a image and description (https://prnt.sc/t5ykcv)
- The details switch to column view on smaller devices. This is the same for all project pages. (https://prnt.sc/t5ykql)

**NVIDIA RC Car**
- Landing view for the RC car page, this page (header title and details div) have parallax scrolling (https://prnt.sc/t5ylgp)
- Details for the NVIDIA Page, there is a video demo that has a fullscreen option (https://prnt.sc/t5yl37)

**Facebook Academy**
- Parallax landing page (https://prnt.sc/t5yltp)
- Details for FB Academy Page. Video demo works (https://prnt.sc/t5ym3z)

**Robotics Page**
- Landing page screenshot failed, but I have a local copy
- Details for Robotics (https://prnt.sc/t5ynaf)

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -12,17 +12,56 @@
...
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+
+$(document).ready(function() {
+
```
Comment (cassiehan) [inline portfolio/src/main/webapp/script.js:17]: remove empty line

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -12,17 +12,56 @@
...
+    // locate gallery div
```
Comment (cassiehan) [inline portfolio/src/main/webapp/script.js:18]: divison

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -12,17 +12,56 @@
...
+    anchor.addEventListener('click', function (e) {
+        e.preventDefault();
+
+        document.querySelector(this.getAttribute('href')).scrollIntoView({
+            behavior: 'smooth'
+        });
+    });
+});
+
+function displayPhoto(img, images, index) {
```
Comment (cassiehan) [inline portfolio/src/main/webapp/script.js:37]: please add comments to each method

**Comment by RickyAvina** [comment]
Merging into master.

Added in contact page with links to social and a message board (https://prnt.sc/t6lcdy)

**Pull request merged**